agent/src/Lib/External/AppManifest.hs

```diff
@@ -91,9 +92,16 @@ data AppManifest where
                     , appManifestStartAlert :: Maybe Text
                     , appManifestActions :: [Action]
                     } -> AppManifest
+deriving instance Show AppManifest
 
-uiAvailable :: AppManifest -> Bool
-uiAvailable AppManifest {..} = isJust $ HM.lookup 80 appManifestPortMapping
+torUiAvailable :: AppManifest -> Bool
+torUiAvailable AppManifest {..} = any (== 80) $ portMapEntryTor <$> appManifestPortMapping
+
+lanUiAvailable :: AppManifest -> Bool
+lanUiAvailable AppManifest {..} = any id $ fmap portMapEntryLan appManifestPortMapping <&> \case
+    Just Standard     -> True
+    Just (Custom 443) -> True
+    _                 -> False
```

Comment:
standard lans yield UI's, also a custom lan with 443 exposed, we may also want to add 80 here?

Reply:
Yeah, add port 80 just in case we need it